pkgs/top-level/all-packages.nix

```diff
@@ -7885,7 +7885,7 @@ with pkgs;
 
   ovito = libsForQt5.callPackage ../applications/graphics/ovito { };
 
-  owncloud-client = libsForQt514.callPackage ../applications/networking/owncloud-client { };
+  owncloud-client = libsForQt515.callPackage ../applications/networking/owncloud-client { };
```

Comment:
```suggestion
  owncloud-client = libsForQt5.callPackage ../applications/networking/owncloud-client { };
```

Reply:
Thank you for your suggested change. I made the change.